backend/setup.cfg

```diff
@@ -6,6 +6,7 @@ version = 0.0.1
 package_dir =
     rope.api = api
     rope.db = db
+    rope.scripts = scripts
```

Comment:
Minor: I think we may need / want to add an empty `backend/scripts/__init__.py`.